crypto/cpuid.c

```diff
@@ -14,7 +14,7 @@
         defined(__x86_64) || defined(__x86_64__) || \
         defined(_M_AMD64) || defined(_M_X64)
 
-extern unsigned int OPENSSL_ia32cap_P[4];
+extern unsigned int OPENSSL_ia32cap_P[5];
```

Comment:
We have prepared PR with ia32cap_P update: https://github.com/openssl/openssl/pull/25709
Depending on the PRs' merge order this part might need an update